spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1094,24 +1094,46 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
             None
           }
 
-        // TODO waiting for arrow-rs update
-//      case RLike(left, right) =>
```

Comment:
I forgot the details. I remember this was added by someone at the early development. Not sure if we have anything update in arrow-rs for it.

Reply:
cc @snmvaughan